14.json

```diff
@@ -83,8 +83,11 @@
       "type": "object",
       "properties": {
         "open": {
-          "description": "A flag which indicates whether the space is currently open or closed. The state 'undefined' can be achieved by omitting this field. A missing 'open' property carries the semantics of a temporary unavailability of the state, whereas the absence of the 'state' property itself means the state is generally not implemented for this space.",
-          "type": "boolean"
+          "description": "A flag which indicates whether the space is currently open or closed. The state 'undefined' can be achieved by omitting this field. A missing 'open' property carries the semantics of a temporary unavailability of the state, whereas the absence of the 'state' property itself means the state is generally not implemented for this space. This field is also allowed to explicitly have the value null for backwards compatibility with older schema versions.",
```

Comment:
Should we add a deprecation note? Endpoints targeting only v14 or higher should not set the field to null.

Reply:
How would the deprecation note look like?

Reply:
Simply stating in the description that the field is deprecated 🙂 (I don't think there's a formal mechanism.)

Reply:
Well the field is not deprecated, just setting it to `null` is, but I guess this is what you meant.

Reply:
I mentioned that it is deprecated.